toolset/setup/linux/prerequisites.sh

```diff
@@ -63,10 +63,16 @@ sudo pip install colorama==0.3.1
 sudo pip install progressbar==2.2
 sudo pip install requests
 
-# Install gcc-4.8, gcc-4.9 and clang
+# Install gcc-4.8, gcc-4.9
 sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
 sudo apt-get -yq update
-sudo apt-get install -qqy gcc-4.8 g++-4.8 gcc-4.9 g++-4.9 clang-3.5
+sudo apt-get install -qqy gcc-4.8 g++-4.8 gcc-4.9 g++-4.9
+
+# Install Clang, needed by silicon.
```

Comment:
Sorry to jump in late on this, but we've got to move the compiler stuff out of prereqs. This is loading once for every test and 158/161 tests don't need it. Most of the C/C++ tests are getting everything they need from the `build-essential` package. It would be great if this could get moved to a `/toolset/setup/linux/systools/` script and brought in as needed. I think only ulib silicon and libreactor are affected by this. @matt-42 I can hopefully do this within the next couple of days unless you want to take a crack at it.